### DISCUSSION TITLE
Support for desktop

### DISCUSSION THREAD

**Discussion by YoussefHachicha** [cashapp/redwood#1753] [General]
When will  it support desktop?

**Comment by JakeWharton** [comment]
Can you be more specific? Are you referring to something on the JVM like AWT, Swing, or Compose UI? Or are you referring to native applications and a native UI toolkit like GTK or Qt?

**Reply by YoussefHachicha** [reply]
I meant JVM, Compose UI

**Reply by JakeWharton** [reply]
Compose UI for JVM desktop is fully supported already.

The counter sample is here: https://github.com/cashapp/redwood/tree/trunk/samples/counter/desktop-composeui

The emoji sample is currently blocked on unrelated libraries: https://github.com/cashapp/redwood/issues/643. Hopefully that will be able to be resolved soon, though.
